GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -143,7 +143,7 @@ void oneTimeShutDown() {
     void beforeDrawEyes() {
         mGVRViewManager.beforeDrawEyes();
     }
-
+    
```

Comment:
Generally shouldn't check-in a file if all you did was change the whitespace.

(The exception would be if the whole point of the check-in was to apply uniform formatting to the whole project.)